assets/src/scss/components/_buttons.scss

```diff
@@ -261,3 +261,27 @@ button.load-more-mt {
     transition: opacity 400ms;
   }
 }
+
+.btn-post-tag {
+  --button-post-tag-- {
```

Comment:
Since this is a new class we're introducing, it could be nicer to just choose a class name that plays well as a variable prefix. To reduce the amount of variations of the name we need to deal with.

Reply:
I'm not sure what you mean here by "plays well as a variable"?

Reply:
I mean you could use `_--` so that it uses the class name which is already on the previous line. I assume you didn't use it because `btn-post-tag` doesn't look nice as a variable name?

Reply:
Oh, no, I think it's fine but other button classes used a separate variable prefix so I did the same 😅 But we could use `_--`

Reply:
[Updated](https://github.com/greenpeace/planet4-master-theme/pull/1630/commits/f35df4cbed49cacfe5f7f962aaf55b65823a03ab)!